gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/http/HttpProxyOptions.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.definition.model.v4.http;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Getter
+@Setter
+public class HttpProxyOptions implements Serializable {
+
+    private static final long serialVersionUID = -1133018497662951240L;
+
+    @JsonProperty("enabled")
```

Comment:
I just wanted to avoid breaking rest api or json schema if someone decide to rename a java attribute in the futur. Having the jsonProperty allows to explicitly see that you need to take care about the consequences you could not think about by "just renaming a java attribute".

Anyway it's more a something to discuss globally before taking a decision so to keep consistency with other v4 classes, I'll remove all the json property annotations 👍